### PR TITLE
[all][fr] Dictionnary additions

### PR DESCRIPTION
**Pull request by LucieSteib**
Adding words to fr dics as per [#5240](https://github.com/languagetooler-gmbh/languagetool-premium/issues/5240).
Adding to spelling_global:
`Scribbr`
`Giuseppe Arcimboldo`

### CODE REVIEW

**Review by jaumeortola — APPROVED** [review]

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/hunspell/spelling.txt [hunk 1]
```diff
@@ -34305,4 +34305,43 @@ Loing
...
+zumba
+L’Énéide
+Arnolphe
+anacoluthon
+Frosties
+Twix
+Lanvin
+Jardiland
+Cracotte
+Croustibat
```
Comment (GillouLT) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/hunspell/spelling.txt:34318]: I can't believe this was missing!

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/hunspell/spelling.txt [hunk 2]
```diff
@@ -34305,4 +34305,43 @@ Loing
...
+Decap'four
+Délichoc
+Monopoly
+Pom'Potes
+Pur'Soup
+Shoko-Bons
```
Comment (GillouLT) [inline languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/hunspell/spelling.txt:34324]: I think the registered name is "Schoko-Bons"

**Review by GillouLT — APPROVED** [review]
Besides schoko-bons, LGTM!

File: languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/hunspell/spelling.txt (hunk 1 above)
Comment (LucieSteib) [reply languagetool-language-modules/fr/src/main/resources/org/languagetool/resource/fr/hunspell/spelling.txt:34318]: Qui peut te battre !

**Pull request merged**